README.md

```diff
@@ -70,7 +70,7 @@ This repo hosts:
 [Make your own badges!][custom badges]
 (Quick example: `https://img.shields.io/badge/left-right-f39f37`)
 
-Browse a [complete list of badges][shields.io].
+Browse a [complete list of badges][shields.io] and find the badge you want to add to your GitHub readmes or other web page. Click on the badge to fill in required data (like server, colors etc.), and its ready for use!
```

Comment:
Thanks for this! There were originally some minor grammar/text changes I was going to suggest, but then I thought it may be beneficial to just expand this a bit more.

What do you think about something like this?

```suggestion
Browse a [complete list of badges][shields.io]. Locate a particular badge by using the search bar or browsing categories. Click on the badge to fill in required data elements for that badge type (like your username or repo) and optionally customize (label, colors etc.). and it's ready for use! Use the button at the bottom to copy your badge url or snippet, which can then be added to places like your GitHub readme files or other web pages.
```

Reply:
I tried to had it short first (in order to not make a lot of change), but in general I am in favour to be detailed and explicit, in order to show how it works step by step. 

Yes, I like the more explicit version. 

Maybe a small title could be added just before that paragraph? Something like

`### Using a badge for your project`

What do you think?

Reply:
Yeah the more I think about this, the more this feels like a potential "Using" or "Quickstart" section of information vs. a line item under the "Examples" heading.

Feel free to update the PR here with a proposal, sure other maintainers will weigh in with their thoughts

Reply:
> Feel free to update the PR here with a proposal, sure other maintainers will weigh in with their thoughts

My main thought is that the Readme is quite long and already has quite a few sections. Have we got actual evidence that users have been struggling to use the website, and if so will adding more text to the Readme really help much? If a fairly simple webpage like shields.io is not intuitive or confusing, I feel like we should capture that in specific issues. Some contributors interested in frontend have recently enquired about areas to improve and we haven't been able to properly direct them, having some concrete pointers to improve would help.

I'm not against this change per say, I'm simply not convinced it will have a positive impact.

Reply:
So I guess there's really two parts to this: would it be helpful for Shields to provide a quickstart/how-to docs for users, and if so, where/how should that be documented?

IMO, it is indeed helpful material. This originated with #5350, however, I've also seen other cases where users were unaware of certain aspects of the Shields.io site, for example being able to open the modal window for customization. I'll also confess that even as a long time user of Shields, I had no idea the badge suggestion feature existed until months after I'd been contributing :smile: We also have recurring cases of folks not finding/understanding the dynamic badges located towards the bottom of the page.

I personally feel like the readme file is actually pretty short, at least relative to a lot of other projects, and the vast majority of the current content is about contributing/developing vs. material for _consumers_ of Shields.io. Regardless of the length perceptions though, I don't think users will typically read top-to-bottom to find a specific piece of information. As long as the material is organized well, things like a navigable table of contents or even basic search could be used to help users get to a specific location. 

I don't have any strong opinions about _where_ any of this should go, be it the simple patch text to the readme, or a Discussion, or in-site, and/or some combination. However, I'd typically rather over communicate than under communicate, and while I know this patch doesn't solve everything, I would prefer moving ahead with some minor improvements if they help vs. waiting for a perfect solution.

Reply:
Okay, let's go with this then! 👍🏻